README.md

```diff
@@ -1,31 +1,29 @@
 # Project #1: About Me

```

Comment:
You forgot to change the name of the project :)
Also, it looks like you have an extra `reset.css` file.